service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/OrderQuestionBlock.java

```diff
@@ -56,7 +56,9 @@
         "manageOrdersExclusionRequirementDetails", "Add exclusion details", OrderSection.ORDER_DETAILS,
         List.of("manageOrdersHasExclusionRequirement", "manageOrdersExclusionDetails")),
     CLOSE_CASE("closeCase", "Closing case", OrderSection.REVIEW,
-        List.of("manageOrdersCloseCase", "manageOrdersCloseCaseWarning"));
+        List.of("manageOrdersCloseCase", "manageOrdersCloseCaseWarning")),
+    WHICH_OTHERS("whichOthers", "Which others", OrderSection.OTHER_DETAILS,
+        List.of("sendOrderToAllOthers", "others_label", "othersSelector"));
```

Comment:
where others_label is coming from?

Reply:
discussed. defined for another event so already there